sdk/search/search-documents/package.json

```diff
@@ -22,7 +22,7 @@
     "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace dist-esm/**/*.spec.js --harmony",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o search-documents-lintReport.html || exit 0",
```

Comment:
why did we add this back? doesn't this mean we won't block CI on lint errors?

Reply:
I was adding this.skip which was causing lint issues. I have changed it to it.skip (and created a seperate file for spec also). Changed it back